cli/src/cmd_auth.rs

```diff
@@ -538,8 +539,62 @@ impl CmdAuthStatus {
 
     fn error_msg(e: &oxide::Error<oxide::types::Error>) -> String {
         match e {
-            oxide::Error::ErrorResponse(ee) => format!("Error Response: {}", ee.message),
-            ee => ee.to_string(),
+            oxide::Error::CommunicationError(error) => {
+                if error.is_timeout() {
+                    "Request timed out".to_string()
+                } else if error.is_connect() {
+                    // Reqwest will just tell us there was an error; we want to
+                    // look at its internal error to understand the cause.
+                    // Regardless this is reflective of a programming error.
```

Comment:
I don't understand this comment, why is a connection error indicative of a programming error?

Reply:
oops; a stale comment